README.md

```diff
@@ -22,10 +22,10 @@ $ symfony-upgrade-fixer fix /path/to/code --dry-run
 | Name  | Description |
 | ----  | ----------- |
 | property_access | Renamed PropertyAccess::getPropertyAccessor to PropertyAccess::createPropertyAccessor. |
+| inherit_data_aware_iterator | The class VirtualFormAwareIterator was renamed to InheritDataAwareIterator. |
```

Comment:
We only change `README.tml`, then generate with `./symfony-upgrade-fixer readme > README.md`.